### PR TITLE
Add monkey's paw tile

### PR DESCRIPTION
**Pull request by ajcoppa**
Shows in-run most useful wishes, marking them gray if they're not currently accessible and removing them from the list if they're not needed anymore. Includes a reference list of all the paw skills.

Screenshot 2023-04-23 at 12 02 08 PM

All feedback welcome, and I'll add some comments inline in specific areas where I could use another look.

### CODE REVIEW

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:6]: Wish I could union theItem and theEffect together but ASH doesn't support this AFAICT.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:133]: I'm guessing this conditional isn't quite right – could use help figuring out when Grops are technically accessible

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:162]: Let me know if there are other commonly-useful in-run wishes we should add.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:172]: We could flesh this out more, but I assume a lot of people are either going to let garbo handle it or have highly custom setups.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:220]: Is having all of these actually useful? We could collapse it down to just the 5/1 skills, or even put the whole thing in a table in a tooltip.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:116]: the mafia wiki says inequality is the "tavern not unlocked" condition, not >; can we just swap this to a not-equal

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:194]: i do not know if there is a good answer for this but has anyone spaded if you can wish GSBs prior to grops existing in the war? i would assume probably not, but if that is the case, you need to have 400+ kills before you can wish a GSB, so the availability constraint would be slightly wrong here

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:208]: probably want to say "8-bit (among other things)" -- didn't really need it for 8-bit but it saved almost 2 turns on nuns.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:217]: as a description: "summon the nagamar"

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:220]: similarly, only recommend this if the user has no lowercase N but -does- have a heavy D, ruby W, and metallic A; sometimes you don't have em all!

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (docrostov) [inline Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:238]: okay my one callout here -- this feels like this tile would be enormously long to users who have unlocked a lot and not finished a lot. should we think through filtering criteria or something that forces the list to be a little more compact, like not showing > 8 wishes or something?

**Review by docrostov — CHANGES_REQUESTED** [review]
requesting some changes but it's pretty close, we should be ready to merge up soon

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [reply Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash:44]: Oops, I ran into some issues with quest state and this was leftover debug I forgot to remove. Will clean up.

File: Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash
Comment (ajcoppa) [reply Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash]: I agree, this gets fairly big quickly. What do you think about addressing that in a separate follow-up PR? I want to play with some options but I'm inclined not to hold up getting the first version of the tile in based on that. The good news is even though it can get big, it's down in the resources section so it's not going to monopolize the top tasks area or anything.

Comment (docrostov) [reply Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash]: that is fine. i'll approve and merge after i skim it in a few mins

**Review by docrostov — APPROVED** [review]
implemented changes look good. mergin' it up

**Pull request merged**
